### PR TITLE
Chemical renaming

### PR DESCRIPTION
**Pull request by efmkoene**
Fixes https://github.com/C2SM/processing-chain/issues/37

This should eventually close https://github.com/C2SM/processing-chain/issues/37 once `async` becomes the main code. It should push back into `async` (I've never pushed a branched branch, so please check if all goes correctly).

### CODE REVIEW

**Comment by mjaehn** [comment]
@efmkoene Thanks for this PR! I will review this one, but will set the base branch to v3.1-rc once async is merged (as it is in its final review stage).

File: cases/icon-art-oem-test/config.yaml [hunk 1]
```diff
@@ -41,6 +41,8 @@ chem:
     nameformat: '%Y%m%d_%H'
     suffix: .grb
     inc: 3
+    chem_in_to_chem_out_remap_tracers:
```
Comment (mjaehn) [inline cases/icon-art-oem-test/config.yaml:44]:
```suggestion
    remap_tracers:
```

File: jobs/prepare_art_oem.py [hunk 2]
```diff
@@ -74,7 +74,11 @@ def main(cfg):
             ds_chem['PS'] = ds_chem['LNPS']
             ds_chem['PS'].attrs = ds_chem['LNPS'].attrs
             ds_chem['PS'].attrs["long_name"] = 'surface pressure'
-            ds_chem['TRCH4_chemtr'] = ds_chem['CH4_BG']
+            # Remapping chemical tracer names
+            if "chem_in_to_chem_out_remap_tracers" in cfg.chem:
+                for chem_in, chem_out in cfg.chem[
+                        'chem_in_to_chem_out_remap_tracers'].items():
```
Comment (mjaehn) [inline jobs/prepare_art_oem.py:80]:
```suggestion
            if "remap_tracers" in cfg.chem:
                for chem_in, chem_out in cfg.chem[
                        'remap_tracers'].items():
```

**Review by mjaehn — CHANGES_REQUESTED** [review]
I would prefer to shorten the variable name.

**Comment by efmkoene** [comment]
This satisfies your review :-)

**Comment by mjaehn** [comment]
launch jenkins

**Pull request merged**